dex/values.yaml

```diff
@@ -76,13 +78,21 @@ config:
     level: ""
 
   web:
-    http: ""
+    bind: 0.0.0.0
+    port: 5556
+    tls:
+      enabled: false
 
   grpc:
-    addr: ""
+    bind: 0.0.0.0
+    port: 5557
+    tls:
+      enabled: true
+      clientAuth: false
```

Comment:
`clientAuthEnabled`, `enableClientAuth` or `authenticateClient` would sound better IMHO.

Reply:
you're right!